src/app/features/routing/routes.ts

```diff
@@ -26,6 +26,11 @@ export const ROUTE_USER_INFO: RouteDescription = {
   permissions: [OBSERVER],
   fallback: "/"
 };
+export const ROUTE_BIRD_INFO: RouteDescription = {
+  path: "/bird-info/",
+  permissions: [UNAUTHORIZED],
+  fallback: "/"
+};
```

Comment:
I think this route should be under observations path to provide ability to navigate with direct links. Let's discuss on sync.